server/Arcadia.Assistant.Calendar/SickLeave/EmployeeSickLeaveActor.cs

```diff
@@ -179,6 +182,7 @@ private void OnSickleaveApproved(SickLeaveIsApproved message)
             if (this.EventsById.TryGetValue(message.EventId, out var calendarEvent))
             {
                 this.EventsById[message.EventId] = new CalendarEvent(message.EventId, calendarEvent.Type, calendarEvent.Dates, SickLeaveStatuses.Approved, this.EmployeeId);
+                sickLeaveNotifications.Tell(message);
```

Comment:
Should not be here, but rather in Persist callback. If it's here, this method will be called each time the actor is restored - so it keeps sending emails on system restart